kubernetes/product-deployment/main.jenkins

```diff
@@ -0,0 +1,286 @@
+pipeline {
+    // agent {label 'pipeline-kubernetes-agent'}
+    agent any 
+    environment {
+        EKS_CLUSTER_REGION = 'us-east-1'   
+        EKS_CLUSTER_NAME = 'testgrid-eks-cluster'
+        RDS_STACK_NAME = 'testgrid-rds-stack'  
+        DB_USERNAME = 'root'
+        DB_PASSWORD = '613296496' 
+    }
+    stages {
+        stage('Clone repo') {
+            steps {
+                script {
+                    properties([
+                        parameters([
+                            separator(name: "PRODUCT_ENVIRONMENT", sectionHeader: "Product Details",
+			separatorStyle: "border-width: 0",
+			sectionHeaderStyle: """
+				background-color: #7ea6d3;
+				text-align: center;
+				padding: 4px;
+				color: #343434;
+				font-size: 22px;
+				font-weight: normal;
+				text-transform: uppercase;
+				font-family: 'Orienta', sans-serif;
+				letter-spacing: 1px;
+				font-style: italic;
+                margin-bottom: 25px;
+			"""),
+                            string(
+                                name: 'product_name',
+                                defaultValue: 'apim',
+                                description: 'Product name',
+                                trim: true
+                            ),
+                            string(
+                                name: 'product_version',
+                                defaultValue: '4.1.0',
+                                description: 'Product version',
+                                trim: true
+                            ),
+                            separator(name: "AWS_ENVIRONMENT", sectionHeader: "AWS cloudformation Details",
+			separatorStyle: "border-width: 0",
+			sectionHeaderStyle: """
+				background-color: #7ea6d3;
+				text-align: center;
+				padding: 4px;
+				color: #343434;
+				font-size: 22px;
+				font-weight: normal;
+				text-transform: uppercase;
+				font-family: 'Orienta', sans-serif;
+				letter-spacing: 1px;
+				font-style: italic;
+                margin-bottom: 25px;
+			"""),
+                            string(
+                                name: 'deployment_script_repo_url',
+                                defaultValue: 'https://github.com/Tharsanan1/APIMCloudFormation',
```

Comment:
shall we remove this value from default as these are folked repos? WDYT @VimukthiPerera

Reply:
+1